app/questionnaire/return_location.py

```diff
@@ -0,0 +1,27 @@
+from dataclasses import asdict, dataclass
+
+
+@dataclass(kw_only=True, frozen=True)
+class ReturnLocation:
+    """
+    Used to store return locations in the questionnaire.
+
+    return_to: The name of the section to return to
+    return_to_block_id: The block_id of the block to return to
+    return_to_answer_id: The answer_id of the answer to return to
+    return_to_list_name: The list_name to return to if the location is associated with a list
+    return_to_list_item_id: The list_item_id to return to if the location is associated with a list
+    """
+
+    return_to: str | None = None
+    return_to_block_id: str | None = None
+    return_to_answer_id: str | None = None
+    return_to_list_name: str | None = None
```

Comment:
Just realised this is a param we didn't go with in the end - no need for `return_to_list_name` its only being used in your tests